src/main/webapp/pages/announcements/details.html

```diff
@@ -27,11 +27,27 @@
                     <h2 class="panel-title" th:text="#{announcement.control.details.edit}"></h2>
                 </div>
                 <div class="panel-body">
-                    <textarea class="form-control input-full announcement-input" type="text" id="message" name="message"
+                    <div class="form-group">
+                        <label for="title" th:text="#{announcement.control.title}">__Title__</label>
```

Comment:
Sorry, I'm going to take this opportunity to ask a couple questions. I noticed there are these variables with underscores. Is this Thymeleaf pre-processing?

Reply:
Do you mean the `__Title__`?  If so that is just me making sure that I do not forget to template them because they look ugly on the page.  Just placeholders.

Reply:
Ya, it exists for the cancel and save buttons too.

Reply:
Yup that is me trying to remind myself not to be forgetful and to add the translations